core/store/src/main/java/uk/gov/gchq/gaffer/store/operation/handler/named/AddToCacheHandler.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2023 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.store.operation.handler.named;
+
+import uk.gov.gchq.gaffer.operation.Operation;
+import uk.gov.gchq.gaffer.store.operation.handler.OperationHandler;
+
+public interface AddToCacheHandler<OP extends Operation> extends OperationHandler<OP> {
+    String getSuffixCacheName();
```

Comment:
Why only a new interface for add to cache and not also remove from cache? Also could have a default method here, as with `GetFromCacheHook`.

Reply:
Coz the Resolvers that read from the cache are Hooks. There was no Handler which read from the caches.